docs/source/user-guide/sql/aggregate_functions.md

```diff
@@ -56,45 +53,6 @@ avg(expression)
 
 - `mean`
 
-### `bit_and`
```

Comment:
These are in the new functions here: https://datafusion.apache.org/user-guide/sql/aggregate_functions_new.html